packages/types/index.d.ts

```diff
@@ -59,10 +74,28 @@ export type MetamaskPolkadotRpcRequest =
     | ConfigureSnapRequest
     | AddPolkadotAssetRequest
     | RemovePolkadotAssetRequest
-    | GetChainHeadRequest;
+    | GetChainHeadRequest
+    | SignPayloadJSONRequest
+    | SignPayloadRawRequest;
 
 type Method = MetamaskPolkadotRpcRequest["method"];
 
+export interface WalletEnableRequest {
+  method: "wallet_enable";
+  params: object[];
+}
+
+export interface GetPluginsRequest {
+  method: "wallet_getPlugins";
+}
+
+export interface SnapRpcMethodRequest {
+  method: string;
```

Comment:
@mpetrunic I moved these types from `adapter` package to `types` because `SnapRpcMethodRequest` is used to define `Window.ethereum` interface. Definitions of this global interface are now the same inside `example` and `adapter` packages.